.github/workflows/smoke-test-daily.yml

```diff
@@ -181,6 +181,7 @@ jobs:
         runs-on: ubuntu-20.04
         permissions:
           contents: read
+        needs: [api-tests]
```

Comment:
Just to confirm, the dependency on the `api-tests` package is because it was agreed to run the api-tests first as a kind of sanity test and then run the rest of the tests in parallel, avoiding the current sequential flow (e2e -> api -> others), isn't it? Now it would be something like:
```
api-tests -> rest of testing
```

Reply:
Cool, thanks for confirming @nigeljamesstevenson! I'll approve this 👍